site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,62 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet phenotype-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/phenotype-codes",
+        "reference": {
+          "reference": "ValueSet/phenotype-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge CodeSystem hp",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge CodeSystem/hp",
+        "reference": {
+          "reference": "CodeSystem/hp"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet hpo-observation-type-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/hpo-observation-type-codes",
+        "reference": {
+          "reference": "ValueSet/hpo-observation-type-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge CodeSystem snomed-ct",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge CodeSystem/snomed-ct",
+        "reference": {
+          "reference": "CodeSystem/snomed-ct"
+        }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition ncpi-phenotype, Base: Observation",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/ncpi-phenotype",
+        "reference": {
+          "reference": "StructureDefinition/ncpi-phenotype"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet ncpi-observation-interpretation",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/ncpi-observation-interpretation",
+        "reference": {
+          "reference": "ValueSet/ncpi-observation-interpretation"
+        }
+      },
```

Comment:
We may not have to create this ValueSet because https://www.hl7.org/fhir/valueset-observation-interpretation.html is already bound to `Observation.interpretation`.

Reply:
So, I did this believing that it was for the sake of clarification and also to restrict our options to those two options in the event that we decide to truly lock them down like we did with Observation.code. Maybe this is a good discussion for others in the group?